test/state/host.cpp

```diff
@@ -76,15 +76,17 @@ uint256be Host::get_balance(const address& addr) const noexcept
 
 namespace
 {
+bool is_eof(bytes_view code) noexcept
```

Comment:
There is `is_eof_container()` in eof.hpp

Reply:
Yup, wanted to refactor before merging.